docs/developer-docs/integrations/t-ecdsa/index.md

```diff
@@ -1,21 +1,37 @@
 # Threshold ECDSA: chain-key signatures
-
 ## Overview
-The Internet Computer implements a novel threshold ECDSA protocol as part of its chain-key signatures toolbox. In this protocol, the private ECDSA key exists only as secret shares held by designated parties, namely the replicas of a threshold-ECDSA-enabled subnet on ICP, and signatures are computed using those secret shares without the private key ever being reconstructed. Each replica of such subnet holds a key share that provides no information on its own (in fact, any set of shares up to a threshold of one third cannot be distinguished from a set of random strings). At least one third of the replicas are required to generate a threshold signature using their respective key shares. Besides the actual threshold signing protocol, chain-key ECDSA also comprises protocols for secure key distributed key generation and periodic key resharing, which are crucial parts of the protocol. This makes chain-key ECDSA signatures much more powerful than any off-the-shelf threshold ECDSA protocol.
+The Internet Computer implements a novel threshold ECDSA protocol as part of its chain-key signatures suite. In this protocol, the private ECDSA key exists only as secret shares held by designated parties, namely the replicas of a threshold-ECDSA-enabled subnet on ICP. Signatures are computed using those secret shares without the private key ever being reconstructed. Each replica of such subnet holds a key share that provides no information on its own. At least one third of the replicas are required to generate a threshold signature using their respective key shares. 
+
+Besides the actual threshold signing protocol, chain-key ECDSA also comprises of protocols for secure key distributed key generation and periodic key resharing. This makes chain-key ECDSA signatures much more powerful than any off-the-shelf threshold ECDSA protocol.
+
+Each canister on a subnet has control over a unique ECDSA public key and can request signatures for this public key. A signature is only issued to the eligible canister. Each canister can only obtain signatures for its own ECDSA keys. 
+
+:::caution
+Canisters do not hold any private ECDSA keys or key shares themselves. Threshold cryptography can help enable functionality in the trust model of a blockchain that would be impossible to achieve with conventional cryptography alone.
```

Comment:
> Canisters do not hold any private ECDSA keys or key shares themselves.

Note that canisters do not have access to their private ECDSA keys. They can only request signatures. This is because the private key is never stored in a single place. It exists as a decentralized set of secret shared distributed over nodes.